account-kit/core/src/actions/createAccount.ts

```diff
@@ -45,9 +45,11 @@ export type AccountConfig<TAccount extends SupportedAccountTypes> =
     ? OmitSignerTransportChain<
         CreateMultiOwnerModularAccountParams<Transport, AlchemyWebSigner>
       >
-    : OmitSignerTransportChain<
+    : TAccount extends "ModularAccountV2"
+    ? OmitSignerTransportChain<
         CreateModularAccountV2Params<Transport, AlchemyWebSigner>
-      >;
+      >
+    : never;
```

Comment:
Just so I understand the old behavior: previously, it would fall into the MAv2 branch as a default branch, and now we're asserting that the user has to provide "ModularAccountV2" as a mode?

Reply:
This change isn't necessary. I was just trying to make it more explicit.

Reply:
It makes sense! Just wanted to clarify